doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -112,19 +112,30 @@ At this time, we do not make any guarantees about the exactness or freshness of
 
 ### `mz_cluster_replica_frontiers`
 
-The `mz_cluster_replica_frontiers` table describes the frontiers of each [dataflow] in the system.
-[`mz_compute_frontiers`](#mz_compute_frontiers) is similar to this table, but `mz_compute_frontiers`
-exposes the frontiers known to the compute replicas while `mz_cluster_replica_frontiers` contains
-the frontiers the coordinator is aware of.
+The `mz_cluster_replica_frontiers` table describes the per-replica frontiers of
+each source, sink, materialized view, index, and subscription in the system,
+as observed from the coordinator.
```

Comment:
This source also contains frontiers of on-off SELECT dataflows. Should we just gloss over those here?

Reply:
I guess strictly speaking these are short-lived indexes, so maybe fine to not go into too much detail.